src/jobsautoreport/report.py

```diff
@@ -226,4 +229,5 @@ def get_report(self, from_date: datetime, to_date: datetime) -> Report:
                 ]
             ),
             total_number_of_machine_leased=len(step_events),
```

Comment:
you can actually count from the usage now, no?

Reply:
If there is perfect correlation between what we see in usages and the number of successful leases we see in steps